src/actions.rs

```diff
@@ -376,6 +381,15 @@ impl Action {
             .set_section_info(change_elder.new_section);
     }
 
+    pub fn get_section_split_votes(&self) -> Vec<ParsecVote> {
```

Comment:
It's true that it's always going to be only 2, however it gets passed to the slightly more general `wait_votes` which is a `Vec` so to avoid the conversion steps I left it as `Vec`